lightning/src/ln/chan_utils.rs

```diff
@@ -1193,6 +1195,7 @@ impl_writeable_tlv_based!(CommitmentTransaction, {
 	(10, built, required),
 	(12, htlcs, vec_type),
 	(14, opt_anchors, option),
+	(16, opt_non_zero_fee_anchors, option),
```

Comment:
alternatively, I can create a `const_false` "TLV" directive for the macros, so that nothing is actually serialized for this field.

Reply:
I am doing `self.opt_non_zero_fee_anchors.is_some()` instead of `self.opt_non_zero_fee_anchors.unwrap_or(false)`, but they are equivalent.

if it was serialized by an old version, the field won't exist, and that will be interpreted as `None`.

so this should work as written